webi-essentials/README.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Webi Essentials
+homepage: https://webinstall.dev/webi-essentials
+tagline: |
+  The Webi Essentials are curl, git, tar, wget, xz, zip, & webi shell integrations
+---
+
+### Cheat Sheet
+
+> Even Webi needs a place to start.
+
+This installs the tools that are commonly needed to download and unpack various
+packages, using the native package manager.
+
+**This requires root or `sudo`**
+
+### Linux (Debian, Ubuntu)
+
+```sh
+sudo apt update
+sudo apt install -y \
+    curl \
+    git \
+    tar \
+    wget \
+    xz \
+    zip
+```
+
+### Alpine (Docker)
+
+```sh
+sudo apk add --no-cache \
+    curl \
+    git \
+    tar \
+    wget \
+    xz \
+    zip
+```
```

Comment:
Are you skipping the `Files` section as this is a meta package?

Reply:
It's really platform dependent, except for `xz`. Webi doesn't manage much for this.

Reply:
I added the Files section anyway. It's darn helpful as an at-a-glance reference for the macOS locations especially.